onedal/covariance/covariance.cpp

```diff
@@ -18,6 +18,10 @@
 
 #include "oneapi/dal/algo/covariance.hpp"
 
+#define NO_IMPORT_ARRAY // import_array called in table.cpp
```

Comment:
Why is this needed?

Reply:
https://github.com/numpy/numpy/issues/9309#issuecomment-311320497
without this define it fails